JapaneseFilter/sections/specific.txt

```diff
@@ -8732,3 +8732,4 @@ adguard.info,adguard.com,adguard.app##.hello_from_adguard_apps
 !+ NOT_PLATFORM(windows, mac, android, ios, ext_ublock) NOT_OPTIMIZED
 adguard.info,adguard.com,adguard.app##.hello_from_adguard_ext
 !
+sakisiru.jp##.newsletter
```

Comment:
please add the domain into `tastesoflizzyt.com##.newsletter` at `AnnoyancesFilter/Popups/sections/popups_specific.txt` and remove the current rule.

Reply:
Done